src/routes/safe/components/Apps/components/ConfirmTransactionModal.tsx

```diff
@@ -110,7 +111,7 @@ export const ConfirmTransactionModal = ({
     gasCostFormatted,
     txEstimationExecutionStatus,
   } = useEstimateTransactionGas({
-    txData: encodeMultiSendCall(txs),
+    txData: txData || '',
     txRecipient: MULTI_SEND_ADDRESS,
```

Comment:
The receipient needs to change. currently it the tx is always send to the multisend address, which is wrong

Reply:
:man_facepalming: :man_facepalming: :man_facepalming: :man_facepalming: :man_facepalming: